### PR TITLE
New amocrm source

### PR DESCRIPTION
**Pull request by grozwalker**
## What
Add Amo CRM source from  https://www.amocrm.ru/developers

## How
Python Http Api for sync leads from AmoCRM.  Working version can be viewed  in docker image grozwalker/airbyte-source-amocrm

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Connector version is set to `0.0.1`
    - [ ] `Dockerfile` has version `0.0.1`
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-amocrm/source_amocrm/schemas/TODO.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# TODO: Define your stream schemas
+Your connector must describe the schema of each stream it can output using [JSONSchema](https://json-schema.org). 
+
+The simplest way to do this is to describe the schema of your streams using one `.json` file per stream. You can also dynamically generate the schema of your stream in code, or you can combine both approaches: start with a `.json` file and dynamically add properties to it.
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-amocrm/source_amocrm/schemas/TODO.md:4]: Remove this file.

File: airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/TODO.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ 
+The schema of a stream is the return value of `Stream.get_json_schema`.
+ 
+## Static schemas
+By default, `Stream.get_json_schema` reads a `.json` file in the `schemas/` directory whose name is equal to the value of the `Stream.name` property. In turn `Stream.name` by default returns the name of the class in snake case. Therefore, if you have a class `class EmployeeBenefits(HttpStream)` the default behavior will look for a file called `schemas/employee_benefits.json`. You can override any of these behaviors as you need.
+
+Important note: any objects referenced via `$ref` should be placed in the `shared/` directory in their own `.json` files.
+ 
+## Dynamic schemas
+If you'd rather define your schema in code, override `Stream.get_json_schema` in your stream class to return a `dict` describing the schema using [JSONSchema](https://json-schema.org).
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/TODO.md:14]: Remove this file.

File: airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/ad_groups.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/ad_groups.json:3]: It is missing the additionalProperties and usually the fields are `["null", "type"]`

File: airbyte-integrations/connectors/source-yandex-direct/.dockerignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+*
+!Dockerfile
+!main.py
+!source_yandex_direct
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-yandex-direct/.dockerignore:4]: Please move this to another PR.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Check my comments looks there is two sources been added in this one.

File: airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/ad_groups.json (hunk 3 above)
Comment (grozwalker) [reply airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/ad_groups.json]: I delete this file from PR^ it from another source.

**Comment by grozwalker** [comment]
> @grozwalker can you sign the CLA?
Yes and I agree licence

**Comment by grozwalker** [comment]
@marcosmarxm up

**Comment by marcosmarxm** [comment]
Hello, our team ended up losing track of reviewing your contribution and we're sorry about that. I'm cleaning up our backlog and closing it now. If you'd like to resubmit it, we'll be very happy and we'll do our best to prioritize the proper review. For new connectors, the project always aims to obtain integration accounts with the implemented service to ensure that the connector works effectively over time. Once again, I apologize, and feel free to send me a message on Slack if you have any further questions.
